src/BookList.js

```diff
@@ -9,15 +9,16 @@ const query = gql`
         id
         title
         author
+        numPages
       }
   }
 `;
 
 const useStyle = makeStyles((theme) =>
   createStyles({
-    title: {
-      marginRight: theme.spacing(1),
-    },
+    root: {
+      width: '60%'
```

Comment:
Why 60%?

Reply:
I remove the width:60% and add a Container of material ui.